pkg/runner/types.go

```diff
@@ -213,6 +213,11 @@ var routes = []routev1.Route{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: benchmarkNs,
 			Name:      fmt.Sprintf("%s-passthrough", serverName),
+			Annotations: map[string]string{
```

Comment:
Per this [documentation]( https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration ), the default strategy is `random`

If not, can you point me to the right documentation?

Reply:
oh, that's right, I got mistaken, switching to random then.